app/src/main/res/values/strings.xml

```diff
@@ -261,9 +261,9 @@
   <!-- AddProfileActivity -->
   <string name="add_profile_activity_label">Add Profile</string>
   <string name="add_profile_title">Add Profile</string>
-  <string name="add_profile_input_name">Name*</string>
-  <string name="add_profile_input_pin">3-Digit PIN*</string>
-  <string name="add_profile_input_confirm_pin">Confirm 3-Digit PIN*</string>
+  <string name="add_profile_input_name">Name</string>
+  <string name="add_profile_input_pin">3-Digit PIN</string>
+  <string name="add_profile_input_confirm_pin">Confirm 3-Digit PIN</string>
```

Comment:
Does this get rid of all cases where we use '*' to denote required fields? If not, could you please also remove the others? It'd be nice just to fix this holistically so that there's one correct pattern to follow moving forward.

Reply:
Hi, all other * cases have been removed. PTAL

Reply:
It isn't, and therefore all occurrences of it have been removed. PTAL